src/pytorch_lightning/utilities/cli.py

```diff
@@ -123,12 +124,10 @@ def _populate_registries(subclasses: bool) -> None:  # Remove in v1.9
     else:
         # manually register torch's subclasses and our subclasses
         OPTIMIZER_REGISTRY.register_classes(torch.optim, Optimizer, show_deprecation=False)
-        LR_SCHEDULER_REGISTRY.register_classes(
-            torch.optim.lr_scheduler, torch.optim.lr_scheduler._LRScheduler, show_deprecation=False
-        )
+        LR_SCHEDULER_REGISTRY.register_classes(torch.optim.lr_scheduler, LRScheduler, show_deprecation=False)
```

Comment:
Use `_TORCH_LRSCHEDULER` here

Reply:
This pull request is for v1.9, right? If so, I think the file `src/pytorch_lightning/utilities/cli.py` should not exist. Maybe either don't change this file, or beforehand merge a pull request that removes this deprecation.

@qmaruf note that this will avoid the error you are having.

Reply:
Good point. Yes, this file can be removed